skyrim-platform/src/platform_se/skyrim_platform/papyrus-vm-lib/ActivePexInstance.cpp

```diff
@@ -11,7 +8,7 @@ ActivePexInstance::ActivePexInstance()
 }
 
 ActivePexInstance::ActivePexInstance(std::shared_ptr<PexScript> sourcePex,
```

Comment:
Let's revert these changes

Reply:
Revert all spelling corrections in papyrus-vm-lib? They are interconnected.